app/Http/Controllers/WorkplacelearningPeriodController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+
+namespace app\Http\Controllers;
+
+// Use the PHP native IntlDateFormatter (note: enable .dll in php.ini)
+
+use App\Category;
+use App\Cohort;
+use App\Repository\Eloquent\CohortRepository;
+use App\Services\CurrentUserResolver;
+use App\Workplace;
+use App\WorkplaceLearningPeriod;
+use Carbon\Carbon;
+use Illuminate\Http\Request;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Collection;
+use Illuminate\Support\Facades\Auth;
+use InvalidArgumentException;
+use Validator;
+
+class WorkPlaceLearningPeriodController extends Controller
```

Comment:
The class and file names are not identical, this can cause issues with autoloading.